pyseir/cli.py

```diff
@@ -48,6 +49,26 @@ def _generate_whitelist():
     gen.generate_whitelist()
 
 
+def _generate_infection_rate_metric(regions: List[pipeline.Region] = None):
+    """
+    Apply infer_rt.run_rt_for_fips for each region in regions and return a combined results table
+    with a unique row for each region+date combination.
+    """
+    if regions is None:
```

Comment:
`if not regions`

Reply:
It's a moot point because I removed that outside this function after talking with Tom, but I thought `if X is None` was how to check for None -> https://www.python.org/dev/peps/pep-0008/#programming-recommendations. Has that been updated somewhere else?

Reply:
I wanted three things here: (1) if regions is None run for all fips, (2) if empty (evaluates to False) then return empty Dataframe, and (3) if not empty run for each item in the list.